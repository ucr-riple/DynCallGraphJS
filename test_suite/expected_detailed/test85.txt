{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test85.js:19:14:19:42)': [
        'Proxy (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test85.js:19:14:19:42)'
    ],
    'system (Native)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test85.js:8:8:17:4)'
    ],
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test85.js:20:1:20:17)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test85.js:13:14:15:8)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test85.js:14:16:14:37)': [
        'apply (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test85.js:14:16:14:37)'
    ],
    'apply (Native) (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test85.js:14:16:14:37)': [
        'method1 (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test85.js:3:12:5:4)'
    ],
    'method1 (Native) (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test85.js:3:12:5:4)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test85.js:8:8:17:4)'
    ]
}