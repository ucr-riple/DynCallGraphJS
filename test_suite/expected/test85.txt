{ '(test85.js:19:14:19:42)': [ 'Proxy (Native)' ],
  'system (Native)':
   [ '(test85.js:8:8:17:4)' ],
  '(test85.js:20:1:20:17)':
   [ '(test85.js:13:14:15:8)' ],
  '(test85.js:14:16:14:37)': [ 'apply (Native)' ],
  'apply (Native)':
   [ '(test85.js:3:12:5:4)' ],
  'method1 (Native)':
   [ '(test85.js:8:8:17:4)' ] }